docs/configuration.rst

```diff
@@ -116,6 +116,9 @@ return code will be:
  * ``1`` if one or more errors occur
  * ``2`` if no errors occur, but one or more warnings occur
 
+If the script is invoked with the ``-q`` (or ``--no-warning``) options, it will
```

Comment:
- The commit names and previous comments talk about `--no-warnings` (with an `s`), I think it's better English (although I'm not a native English speaker) and more coherent with other options.

- I'd prefer reserve `-q` for later use, moreover the letter `q` would match `--quiet`, but no direct link with `--no-warning`. Can you use only `--no-warnings`? (Same for variable names `quiet` in other files)

Reply:
You're right, I've missed the `s` on the doc file, whereas everywhere else I've used `warnings`.

I've also removed `-q` and kept only `--no-warnings`.